lighthouse-core/computed/manifest-values.js

```diff
@@ -43,14 +43,12 @@ class ManifestValues {
         id: 'fetchesIcon',
         failureText: 'Manifest icon failed to be fetched',
         validate: (manifestValue, errors) => {
-          const failedToFetchIconErrors = [
-            // kCannotDownloadIconMessage
-            'Could not download a required icon from the manifest',
-            // kNoIconAvailableMessage
-            'Downloaded icon was empty or corrupted',
+          const failedToFetchIconErrorIds = [
+            'cannot-download-icon',
+            'no-icon-available',
           ];
           return icons.doExist(manifestValue) &&
-            !failedToFetchIconErrors.some(error => errors.includes(error));
+            !errors.some(error => failedToFetchIconErrorIds.includes(error.errorId));
```

Comment:
sweet, this works out even more nicely for us :)

Reply:
yea i agree